libs/components/layout/src/workspace-sidebar/activities/activities.tsx

```diff
@@ -64,43 +64,46 @@ export const Activities = () => {
     const [recentPages, setRecentPages] = useState([]);
     const userId = user?.id;
 
-    /* temporarily remove:show recently viewed documents */
-    const fetchRecentPages = useCallback(async () => {
+    /* show recently edit documents */
+    const getRecentEditPages = useCallback(async () => {
         if (!userId || !currentSpaceId) {
             return;
         }
-        const recent_pages = await services.api.userConfig.getRecentPages(
-            currentSpaceId,
-            userId
-        );
-        setRecentPages(recent_pages);
-    }, [userId, currentSpaceId]);
+
+        const RecentEditPages =
+            (await services.api.userConfig.getRecentEditedPages(
+                currentSpaceId
+            )) || [];
+
+        setRecentPages(RecentEditPages);
+    }, [currentSpaceId, userId]);
 
     useEffect(() => {
         (async () => {
-            await fetchRecentPages();
+            await getRecentEditPages();
         })();
```

Comment:
no need to wrap with iife

Reply:
The specification of asynchronous functions, otherwise there will be a warning

Reply:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/25152247/182829038-9cc09efd-a0ca-4f9d-a12b-eafa85792047.png">
no warning